src/aliases.js

```diff
@@ -50,13 +50,13 @@ const ALIASES = {
         macOpenCmdTemplate: `open -n -a "{{{path}}}" --args /dev/null open {{{pageUrl}}} {{{cmd}}}`
     },
 
-    'edge-chromium': {
-        nameRe:             /edge\s*beta/i,
+    'edge': {
+        nameRe:             /edge/i,
         cmd:                '--new-window',
         macOpenCmdTemplate: 'open -n -a "{{{path}}}" --args {{{pageUrl}}} {{{cmd}}}'
     },
 
-    'edge': {
+    'mse-legacy': {
```

Comment:
Looks like Edge Legacy is the official name of the old Edge now, so why we can't use it as an alias?